sky/resources.py

```diff
@@ -18,6 +21,8 @@
 from sky.provision.gcp import constants as gcp_constants
 from sky.provision.kubernetes import utils as kubernetes_utils
 from sky.skylet import constants
+from sky.skylet.constants import CATALOG_DIR
+from sky.skylet.constants import CATALOG_SCHEMA_VERSION
```

Comment:
Style: Import module rather than specific functions/vars
```
from sky.skylet import constants as skylet_constants
```
and later on use `skylet_constants.CATALOG_DIR` / `skylet_constants.CATALOG_SCHEMA_VERSION`